src/nakamoto-signer-developer-guide.md

```diff
@@ -0,0 +1,196 @@
+# What is this
+
+Draft documentation for sBTC Signer specifically for the Nakamoto [release](https://stacks-network.github.io/sbtc-docs/sbtc-roadmap.html)
+
+The readers of this are assumed to be anyone who is looking to run a sBTC Signer with minimal custom configuration.
+
+# Disclaimer
+
+Development is on going and changes to the Signer configuration should be expected. This document will be updated every sprint to ensure its accuracy.
+
+## Prerequisites
+
+Rust - [To install](https://www.rust-lang.org/tools/install).
+
+Accessible Stacks node - A Stacks node running the stackerDB instance which is used for signer communication and transaction monitoring and broadcasting.
+
+Accessible Bitcoin node - A Bitcoin node used for transaction monitoring and broadcasting.
+
+A Stacks Address - Identify your address as a Signer
+
+## Installing
+
+### Building from Source
+
+If you wish to compile the default binary from source, follow the steps outlined below. Otherwise, download the binary directly (below)
+
+1. First, clone the Stacks sBTC mono repository:
+
+```console
+git clone git@github.com:stacks-network/stacks-blockchain.git
+```
+
+2. Next, navigate to the stacks-signer directory:
+
+```console
+cd stacks-blockchain/stacks-signer
+```
+
+3. Checkout the appropriate release branch you wish to use if you are not using the default main branch
```

Comment:
I think stacks-network actually uses master, not main. So should update to that.

Reply:
fixed 97695ec9ce37137f9836f3e62bd22c7701fb51de